package_managers/bootstrap_image.bzl

```diff
@@ -0,0 +1,164 @@
+#Copyright 2017 Google Inc. All rights reserved.
+
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+
+#     http://www.apache.org/licenses/LICENSE-2.0
+
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Rule for bootstrapping an image from using download_pkgs and install_pkgs """
+
+load("//store:store_provider.bzl", "store_provider")
+load("@io_bazel_rules_docker//docker:docker.bzl", "docker_build")
+
+load("//package_managers:download_pkgs.bzl", "download_pkgs")
+load("//package_managers:install_pkgs.bzl", "install_pkgs")
+
+PACKAGES_FILE_NAME = "packages.tar"
+# Load all the stores get and put.
+load("//store/git:git.bzl",
+     "git_store_get",
+     "git_store_put",
+     _git_store_tools = "tools")
+
+def _impl(ctx):
+    store_key = "{0}/{1}".format(ctx.attr.date, PACKAGES_FILE_NAME)
+ 
+    get_status = git_store_get(
```

Comment:
The way the exit status gets plumbed through is a bit unusual. Would it be possible to move the call to git_store_get all the way into the script below?